js/mapfilter/map_pane/map_pane.js

```diff
@@ -30,6 +30,8 @@ module.exports = require('backbone').View.extend({
 
     this.appView = options.appView
 
+    this.config = options.config
```

Comment:
This gets passed in from `js/mapfilter/appview.js`.  If it didn't can always access it via window.config.  Prefer the direct approach, though.